app/views/shared/_callout.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="row">
```

Comment:
what do you think about naming this partial sth other than `_callout`, idk what would be a good name maybe like `_covid_notice` or sth like that, `_callout` could be confusing a bit 🤔

Reply:
it's a callout, as in a https://cpratt.co/twitter-bootstrap-callout-css-styles/

might be that we will reuse it in the future for something other than covid is what I was thinking

Reply:
sure, it is using the callout but we might have other partials that uses callout as well right?

Reply:
Hmmm, like where? I am not thinking of another place where this would go. 

I do think calling it `_covid_notice` is destining it to be renamed or removed.

If we end up using it more than one place I'd reach for a View Component called "callout" I think, but, to me that's not needed yet

Reply:
sth like just `_notice` would be fine too, but we can leave it as is 👍

Reply:
I like callout because it's semantic with the UI component fwiw